plugins/catalog-backend-module-ldap/src/ldap/client.ts

```diff
@@ -44,9 +46,20 @@ export class LdapClient {
     bind?: BindConfig,
     tls?: TLSConfig,
   ): Promise<LdapClient> {
+    const secureContext =
+      tls && tls.certs && tls.keys
+        ? tlsLib.createSecureContext({
+            cert: readFileSync(tls.certs).toString(),
```

Comment:
Can we use `readFile` from `fs/promises` instead? Little unfortunate to block here, even if it's not a hot path.

Reply:
Done